### PR TITLE
GH Action to post release diff on release PR

### PR DESCRIPTION
**Pull request by anitacaron**
The same as we have in [CL](https://github.com/obophenotype/cell-ontology/blob/master/.github/workflows/post-release-diff.yaml)

### CODE REVIEW

**Review by matentzn — DISMISSED** [review]
I didnt think about this deeply, but LGTM!

File: .github/workflows/post-release-diff.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Prepare release diff comment
+        env:
+            GITHUB_SHA: ${{ github.sha }}
+        run: "echo \"[Here's a diff of how this release impacts uberon-base.owl](https://github.com/obophenotype/uberon/blob/${{ env.GITHUB_SHA }}/src/ontology/diffs/uberon-diff.md)\" >comment.md"
+      - name: Post comment
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: NejcZdovc/comment-pr@v1.1.1
```
Comment (gouttegd) [inline .github/workflows/post-release-diff.yml:26]: Likewise, could you update to `v2`?

**Review by gouttegd — DISMISSED** [review]
All good, but please consider updating the actions we are using (and we should also at some point do that for all our workflows).

File: .github/workflows/post-release-diff.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  post_diff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply .github/workflows/post-release-diff.yml]: Yeah, thanks for the reminder!

File: .github/workflows/post-release-diff.yml (hunk 1 above)
Comment (anitacaron) [reply .github/workflows/post-release-diff.yml]: Yes, done.

**Review by matentzn — APPROVED** [review]

**Comment by anitacaron** [comment]
I cancelled the QC, which was unnecessary for this PR.

**Pull request merged**